src/backends/mysql/statement.cpp

```diff
@@ -241,6 +241,10 @@ mysql_statement_backend::execute(int number)
                     }
                     else
                     {
+			if(rowsAffectedBulkTemp == -1)
+			{
+				rowsAffectedBulkTemp = 0;
```

Comment:
Looks like something wrong with formatting... Perhaps tabs instead of spaces. https://github.com/SOCI/soci/wiki/Style